pkg/invariantlibrary/allowedbackenddisruption/matches_test.go

```diff
@@ -301,3 +306,26 @@ func TestDisruptionDataFileParsing(t *testing.T) {
 	assert.NoError(t, err)
 
 }
+func currentReleaseFromMap(releasesInQueryResults map[string]bool) string {
+	var releaseSlice []struct {
+		Key   string
+		Value bool
+	}
+	for key, value := range releasesInQueryResults {
+		releaseSlice = append(releaseSlice, struct {
+			Key   string
+			Value bool
+		}{Key: key, Value: value})
+	}
+
+	// Sort the slice in descending order
+	sort.Slice(releaseSlice, func(i, j int) bool {
+		return releaseSlice[i].Key > releaseSlice[j].Key
+	})
```

Comment:
Sort isn't going to work if we ever make it to 4.100 :)  But given the above I don't think this function is needed anymore.

Reply:
Discussed a helper function that could be used here and above that converted the versions and used int comparisons.

Reply:
`return historicaldata.GetMajor(releaseSlice[i].Key) > historicaldata.GetMinor(releaseSlice[j].Key)` doesn't look right to me.  We are comparing the Major of i to the Minor of j?  Seems we need to compare both Majors first the Minors.

Reply:
Added a helper function instead of GetMajor, GetMinor